### PR TITLE
Add books - #1x28c

### PR DESCRIPTION
**Pull request by schwartzadev**

### CODE REVIEW

**Comment by gdude2002** [comment]
The format used for the JSON file has changed.

Where you'd get a single `url` before, each resource now has a `urls` attribute that is a list. It looks like this:

```json
"urls": [
  {
    "title": "Website",
    "url": "https://codecombat.com/",
    "icon": "regular/link"
  },
  {
    "title": "GitHub",
    "url": "https://github.com/codecombat/codecombat",
    "icon": "branding/github"
  }
],
```

The icons are font-awesome icons, and you can use any icons from font-awesome 5 (free or pro). A full list of these icons can be found [here](https://fontawesome.com/icons).

Please note that icons are given in the form `style/name`. Valid styles include `branding`, `regular`, `light` or `solid`. Font-awesome icons are varied based on the style you pick, so check which one you need from the icon list above!

File: static/resources.json [hunk 1]
```diff
@@ -155,6 +139,144 @@
...
+            "title": "GitHub",
+            "url": "https://github.com/bslatkin/effectivepython",
+            "icon": "branding/github"
+          }
+        ],
+        "payment": "paid"
+      },
+      "A Byte of Python": {
+        "description": "A free book on programming using the Python language. It serves as a tutorial or guide to the Python language for a beginner audience. If all you know about computers is how to save text files, then this is the book for you.",
+        "urls": [
```
Comment (gdude2002) [inline static/resources.json:206]: There is a physical version that can be bought at https://www.swaroopch.com/buybook/

**Review by gdude2002 — CHANGES_REQUESTED** [review]
Overall a good set of changes. I would like to comment that the description key here is to describe the book itself - who it's for, what it covers, and so on. `"Clear, Concise and Effective Programming"` comes off as more of like a catchphrase. Show that you know what the book is about.

File: static/resources.json [hunk 2]
```diff
@@ -155,6 +139,144 @@
       }
     }
   },
+  "Books": {
+    "description": "The best books for learning Python or Python Frameworks",
+    "resources": {
+      "Automate the Boring Stuff with Python": {
```
Comment (lemonsaurus) [inline static/resources.json:145]: There's also a Udemy course for this. Can we add that too? 

Here's a link with a coupon added that is always going to cost 10 dollars.

https://www.udemy.com/automate/?couponCode=FOR_LIKE_10_BUCKS

**Review by lemonsaurus — CHANGES_REQUESTED** [review]
if you address mine and gdudes comments, I think we're ready to merge.

**Comment by lemonsaurus** [comment]
@schwartzadev Just making sure you noticed that we requested some changes.

**Comment by gdude2002** [comment]
It's been 15 days since there was any activity on this PR. Are you going to address our changes, @schwartzadev?

**Comment by lemonsaurus** [comment]
if we don't see an answer to this today, I vote we close this PR and just finish this task ourselves. I've been missing this change for weeks. I will ping Andrew on discord as well.

**Comment by lemonsaurus** [comment]
Still no word from Andrew on this.

**Comment by lemonsaurus** [comment]
Closing PR.

**Comment by schwartzadev** [comment]
Sorry y'all - I've been busy and won't be able to work on this for a while.
Good luck and my apologies.

**Comment by lemonsaurus** [comment]
that's okay, @schwartzadev. We took care of this ourselves. See you around!

**Pull request closed without merging**